### DISCUSSION TITLE
Add MacOS Shortcuts

### DISCUSSION THREAD

**Discussion by semenovfilipp** [elementary/triage#461] [Q&A]
Hi there,
In Russia me and my friends use ElementaryOS as alternative of macOS on Macbook older than 2016 because a lot of applications especially Homebrew doesn't work on 10.14 macOS.
______
So the main pain of this if shortcuts.
There is a good solution but not all users can easily solve it. It named Kinto sh
https://github.com/rbreaves/kinto
_____
Your OS is looks like macOS and most mac users come to you because of many features that comes from "box".
So please add quick switch to shortcuts or if it has already - add manual on main page of your website and when first load starts.
Thanks

**Comment by lenemter** [comment]
Closing as duplicate of https://github.com/elementary/switchboard-plug-keyboard/issues/435
